# Std. metadata for higher-level flow data types
std_msgs/Header header
# Define module specific info
# Intake image for processing
sensor_msgs/Image image
# Object detection info to broadcast
vision_msgs/Detection2D detection_msg